Ошибка: HTTPConnectionPool(host='84', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000002027AB916C0>: Failed to resolve '84' ([Errno 11001] getaddrinfo failed)"))